config.md

```diff
@@ -108,16 +108,18 @@ For more information about SELinux, see  [Selinux documentation](http://selinuxp
 
 The user for the process is a platform-specific structure that allows specific control over which user the process runs as.
 
-#### Linux User
+#### Linux and Solaris User
 
-For Linux-based systems the user structure has the following fields:
+For Linux and Solaris based systems the user structure has the following fields:
 
 * **`uid`** (int, required) specifies the user id.
 * **`gid`** (int, required) specifies the group id.
 * **`additionalGids`** (array of ints, optional) specifies additional group ids to be added to the process.
 
 _Note: symbolic name for uid and gid, such as uname and gname respectively, are left to upper levels to derive (i.e. `/etc/passwd` parsing, NSS, etc)_
 
+_Note: For Solaris, uid and gid specify the uid and gid of the process inside the container and need not be same as in the host._
```

Comment:
This is also true for Linux.  Instead of this note, I've filed #412 to make this explicit for both platforms.